.travis.yml

```diff
@@ -22,12 +22,17 @@ script:
 - py.test --cov --ignore=test/
 - pip install -r requirements.test.txt
 - ${DIST_REPO}/scripts/package.sh "redhat" "7"
+- ${DIST_REPO}/scripts/package.sh "ubuntu" "14.04"
+- sudo chown -R travis:travis ${DIST_REPO}/rpms/build
+- sudo chown  travis:travis ${DIST_REPO}/deb_dist/*.deb
```

Comment:
Does it make sense to do this instead:

sudo chown -R travis:travis ${DIST_REPO}/